### ISSUE TITLE
`jira issue create` not interactive by default

### ISSUE THREAD

**Issue by JosXa** [ankitpokhrel/jira-cli#737]
**Describe the bug**

On Windows PowerShell:
```
.\jira.exe issues create
✗ Params `--summary` and `--type` is mandatory when using a non-interactive mode
```

I was expecting to see an interactive CLI when not using any parameters as indicated by the README.

**Please provide following details**
1. JiraCLI Version:
```
(Version="1.5.1", GitCommit="0e0f82d52ef5a775d2cc662fbe38d9732b9b5c59", CommitDate="2024-01-20T10:48:06+00:00", GoVersion="go1.21.5", Compiler="gc", Platform="windows/amd64")
```
2. Are you using Jira cloud or on-premise jira server?
```
SERVER INFO
-----------

Version:         1001.0.0-SNAPSHOT
Build Number:    100251
Deployment Type: Cloud
Default Locale:  en_US
```
3. What operating system are you using? Also mention version.
```
Windows 10, Version 10.0.19045 Build 19045
```

4. What terminal are you using? Also mention version.

```
    Windows Terminal
```

**To Reproduce**

1. Download release
2. Open powershell in bin folder
3. Execute `.\jira.exe issue create`

**Expected behavior**
An interactive CLI

**Comment by beatbrot** [comment]
@JosXa This was fixed by me with #706. Most likely, it will be included in the next release :)

**Comment by JosXa** [comment]
Thank you for the work @beatbrot  :)
